backend/controller/controller.go

```diff
@@ -1189,6 +1203,26 @@ func (s *Service) ResetSubscription(ctx context.Context, req *connect.Request[ft
 	return connect.NewResponse(&ftlv1.ResetSubscriptionResponse{}), nil
 }
 
+func dsnSecretKey(module string, db string) string {
+	return fmt.Sprintf("FTL_DSN_%s_%s",
+		strings.ToUpper(stripNonAlphanumeric(module)),
+		strings.ToUpper(stripNonAlphanumeric(db)),
```

Comment:
Modules and database IDs are always identifiers, so this should be unnecessary.

Reply:
That is there to just get rid of underscores, which are allowed in IDs it seems, and break the env var encoding (underscore in module, especially)